packages/ui-app/src/InputError.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017-2018 @polkadot/ui-app authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { BareProps } from './types';
+
+import React from 'react';
+import Label from 'semantic-ui-react/dist/commonjs/elements/Label/Label';
+
+import classes from './util/classes';
+
+type Props = BareProps & {
+  label?: any
```

Comment:
`React.ReactNode` (tyr to steer clear from any as much as possible - yes, it does happen that they are needed)